docs/components/modeler/desktop-modeler/plugins/plugins.md

```diff
@@ -4,15 +4,39 @@ title: Plugins
+You can plug into the modeler to change its appearance, slot React components into our UI, add new menu entries or extend the modeling tools for [BPMN](https://github.com/bpmn-io/bpmn-js), and [DMN](https://github.com/bpmn-io/dmn-js).
+
+Adding a plugin is as easy as putting it into the `resources/plugins` directory relative to your [`{APP_DATA_DIRECTORY}`](../search-paths#app-data-directory) or [`{USER_DATA_DIRECTORY}`](../search-paths#user-data-directory) directory.
 
 The Camunda Modeler searches for available plug-in entry points via the `resources/plugins/*/index.js` glob. That being said, each plug-in must reside in it's own folder which is a direct child of the `plugins` directory. If you download and extract plugins from GitHub the extracted directory contains the actual plugin, so make sure to copy the plugin, not its parent directory.
 
-So let's dive into how to add your own plugins.
+<br>
+
+## Overview of your possibilities as a plugin developer
+
+There are many way you as a developer are able to extend our modeling tools, and since this ecosystem is a little large here's an overview of the ways you can customize our software:
+
+| Plugin type               | Functionality                                                                                                        | Example                                                                                                                         |
+| ------------------------- | -------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| Menu Entries              | Add new entries to the menu bar - useful to interact with your plugins, link to external pages, add settings, ect... | &nbsp;&nbsp;&nbsp;[**↗**](https://github.com/camunda/camunda-modeler-plugins/tree/master/menu-plugin-example)                     |
+| Custom Styles             | Change the look and feel of Camunda Modeler and your extensions                                                      | &nbsp;&nbsp;&nbsp;[**↗**](https://github.com/camunda/camunda-modeler-plugins/tree/master/style-plugin-example)                    |
+| React Components          | Embed custom React components into specific anchor points in our application                                         | &nbsp;&nbsp;&nbsp;[**↗**](https://github.com/pinussilvestrus/camunda-modeler-autosave-plugin)                                     |
```

Comment:
Sometimes you talk about `our application` sometimes about `Camunda Modeler`. Please make it consistent. I recommend:
```suggestion
| React Components          | Embed custom React components into specific anchor points of the Camunda Modeler                                       | &nbsp;&nbsp;&nbsp;[**↗**](https://github.com/pinussilvestrus/camunda-modeler-autosave-plugin)                                     |
```

This also applies to the other table entries